docs/docs-components/data/types.js

```diff
@@ -12,3 +12,64 @@ export type DesignOverview = {|
   path: string,
   svg: SVGElement,
 |};
+
+type StatusType = 'notAvailable' | 'partial' | 'planned' | 'ready';
+
+/**
+ * All components should be in a single list. That list can be filtered by category/platform/etc where needed.
```

Comment:
This is my recommended strategy for refactoring the actual component data. We don't need multiple lists — we already use a `category` field, which allows for any necessary filtering. A single list makes components easier to deal with.